src/main/java/reposense/report/RepoCloner.java

```diff
@@ -230,6 +241,12 @@ private boolean spawnShallowPartialCloneProcess(RepoConfiguration config) {
      */
     private boolean waitForCloneProcess(RepoConfiguration config) {
         try {
+            Path repoPath = Paths.get(FileUtil.REPOS_ADDRESS, config.getRepoFolderName(), config.getRepoName());
+
+            if (SystemUtil.isTestEnvironment() && Files.exists(repoPath)) {
+                return true;
```

Comment:
Doesn't this mean that the repo is already cloned? Then why are we returning true for `wait`?

Reply:
Yes, the repo is already cloned at this stage. The `true` is to indicate that the function is successful, otherwise it will return `false` if an exception has occurred.

Reply:
@Tejas2805

Reply:
@Tejas2805 Ping